FreeTubeApp/FreeTube
# Search: Add hashtags to results

# Title

## Pull Request Type

## Related issue
https://github.com/FreeTubeApp/FreeTube/pull/3483#issuecomment-1537516380

## Description
This PR parses hashtags in the search results

## Screenshots
![image](https://github.com/FreeTubeApp/FreeTube/assets/78101139/9e3b69c7-1961-4eb0-89c5-8bf2162a7fbb)

## Testing 
(Test with both Invidious and local api)
- Search `#hi`, see hashtag in results
- Search `#meow` see updated results + updated hashtag
- Search '#ThisShouldNotAddAHashtagResult'
- 
## Desktop
- **OS:** Windows
- **OS Version:** 11
- **FreeTube version:** 0.18.0

## Additional context
Requires https://github.com/iv-org/invidious/pull/3989 to be merged and https://github.com/LuanRT/YouTube.js/pull/441 to be released
